documentation/shared/attributes.adoc

```diff
@@ -25,6 +25,8 @@
 // Kafka upgrade attributes used in kafka upgrades section
 :DefaultKafkaVersion: 3.6.0
 :DefaultKafkaMetadataVersion: 3.6
+:KafkaMetadataVersionLower: 3.5
+:KafkaMetadataVersionHigher: 3.6
```

Comment:
I think we should use the IV versions here ... we use them in the examples as well because without it patch version downgrades might be possible in some edge cases.

```suggestion
:KafkaMetadataVersionLower: 3.5-IV2
:KafkaMetadataVersionHigher: 3.6-IV2
```